matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/session/space/SpaceCreationTest.kt

```diff
@@ -57,11 +55,9 @@ class SpaceCreationTest : InstrumentedTest {
         val topic = "A public space for test"
         var spaceId: String = ""
         commonTestHelper.waitWithLatch {
-            GlobalScope.launch {
```

Comment:
Since you are not using GlobalScope anymore, `@Suppress("EXPERIMENTAL_API_USAGE")` can be removed. There are maybe other places in the PR.